apps/omg_api/lib/fees.ex

```diff
@@ -23,13 +23,13 @@ defmodule OMG.API.Fees do
 
   require Utxo
 
-  @type fee_spec_t() :: %{token: Crypto.address_t(), flat_fee: non_neg_integer}
-  @type token_fee_t() :: %{Crypto.address_t() => non_neg_integer}
+  @type fee_spec_t() :: %{token: Transaction.currency(), flat_fee: non_neg_integer}
+  @type fee_t() :: %{Transaction.currency() => non_neg_integer} | :ignore
 
   @doc """
   Parses provided json string to token-fee map and returns the map together with possible parsing errors
   """
-  @spec parse_file_content(binary()) :: {list({:error, atom()}), token_fee_t()}
+  @spec parse_file_content(binary()) :: {list({:error, atom()}), fee_t()}
```

Comment:
I have an idea to rearrange function in this file to make it little more readable.
Let's talk